src/module.c

```diff
@@ -7883,6 +7898,12 @@ void moduleReleaseGIL(void) {
  * so notification callbacks must to be fast, or they would slow Redis down.
  * If you need to take long actions, use threads to offload them.
  *
+ * Moreover, the fact that the notification is executed synchronously means
+ * that the notification code will be executed in the middle on Redis logic
+ * (commands logic, eviction, expire). Changing the key space while the logic
+ * runs is dangerous and discouraged. In order to react to key space events with
+ * write actions, please refer to 'RM_AddPostNotificationJob'.
```

Comment:
this should be done with backticks
```suggestion
 * write actions, please refer to `RM_AddPostNotificationJob`.
```